lua/nvim_lsp/metals.lua

```diff
@@ -58,6 +64,26 @@ configs[server_name] = {
     cmd = {bin_name};
     filetypes = {"scala"};
     root_dir = util.root_pattern("build.sbt", "build.sc", "build.gradle", "pom.xml");
+    message_level = vim.lsp.protocol.MessageType.Log;
+    init_options = {
```

Comment:
Up above where we have `--java-opt -Dmetals.client=coc.nvim`, that is used to set base functionality for metals. This was before we implemented anything for `initializationOptions` or `clientExperimentalCapabilities`. Now everything can just be set via `initializationOptions`, which below reflects.